### ISSUE TITLE
intuitive way to see the total power consumption / generation

### ISSUE THREAD

**Issue by Kezii** [electricitymaps/electricitymaps-contrib#4820]
it should be useful to have an intuitive way to see the total power consumption / generation in GW, for example with a circle in the middle of the country/zone with a size proportional to the power, to see at a glance who is producing more pollution with the corresponding co2

**Comment by VIKTORVAV99** [comment]
I don't think we want to add anything directly to the map right now but there has been talks about showing on the individual zone pages in a better way. But it might be possible to add it as a layer similar to solar and wind in the future.

>**Note**
>I am going to convert this to a discussion as this is not really a issue in it self.
